R/utils.R

```diff
@@ -64,6 +64,16 @@ pairwise.sum.decreasing = function(x, y) {
 #'
 #' @export
 panel.matrices = function(panel, unit = 1, time = 2, outcome = 3, treatment = 4) {
+  keep = c(unit, time, outcome, treatment)
```

Comment:
something got duplicated when you merged this, now we have "stop("Column identifiers should be either integer or column names in `panel`.") done twice

Reply:
just restore this function to the master version?